pages/services/[ware].js

```diff
@@ -18,12 +17,14 @@ const Service = () => {
           <ItemPage
             title={ware.name}
             description={ware.description || ware.snippet}
+            id={ware.id}
             img={
               ware.urls.promo_image ? {
                 src: ware.urls.promo_image,
                 alt: `The promotional image for ${ware.name}`,
               } : DEFAULT_WARE_IMAGE
             }
+            slug={ware.slug}
```

Comment:
I wonder if a better refactor can be done here to pass the entire ware instead of breaking down all of its parts? that way if we decide in the future to show more things on the item page, we only have to change it in the library.

Reply:
if we refactor, we should leave the `img` prop though so clients can set their own default images and default img styling.

Reply:
i thought about doing this but wasn't sure if it would be better or not, specifically because of the image prop- i don't mind refactoring though!

Reply:
good deal. let's refactor! except for the img. 🙂

Reply:
done